server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/SetMessagesCreationProcessorTest.java

```diff
@@ -129,7 +131,8 @@ public void setUp() throws MailboxException {
         outbox = mock(MessageManager.class);
         when(outbox.getId()).thenReturn(OUTBOX_ID);
         when(outbox.getMailboxPath()).thenReturn(new MailboxPath(NAMESPACE, USER, OUTBOX));
-        when(outbox.appendMessage(any(InputStream.class), any(Date.class), any(MailboxSession.class), any(Boolean.class), any(Flags.class))).thenReturn(MessageUid.of(1));
+        when(outbox.appendMessage(any(InputStream.class), any(Date.class), any(MailboxSession.class), any(Boolean.class), any(Flags.class)))
+            .thenReturn(new ComposedMessageId(outbox.getId(), new DefaultMessageId(outbox.getId(), MessageUid.of(1)), MessageUid.of(1)));
```

Comment:
MessageUid.of(1) is repeated two times, making this harder to read.

Is it the same ? (If so we might have duplication in the API used). Naming these parameters might helps to read this better.